rc/trafficserver.service.in

```diff
@@ -21,6 +21,9 @@ After=syslog.target network.target
 
 [Service]
 Type=simple
+ExecStartPre=-/usr/bin/mkdir -p /@exp_runtimedir@
+ExecStartPre=/usr/bin/chown @pkgsysuser@:@pkgsysgroup@ /@exp_runtimedir@
```

Comment:
This needs to be a run time value vs a compile value. The user and group can be changed dynamically via a config value and via traffic_layout. It will be more useful to have a $TM_USER and $TM_GROUP value here that can be set independently of the build.

Reply:
I am simply copying what is done in the sysVinit script equivalent, line 180-184 in rc/trafficserver.in appears to do this at compile time as well. 

I don't see any appearances of a TM_USER variable in the source
```

if [ ! -d $TS_BASE@exp_runtimedir@ ]
then
    mkdir -p $TS_BASE@exp_runtimedir@
    chown @pkgsysuser@:@pkgsysgroup@ $TS_BASE@exp_runtimedir@
fi
```

Reply:
could you add these values to trafficserver.in file in the same directory?

This allows the value to be overridden at runtime while providing useful defaults

Reply:
This is specific to systemd cleaning stuff out, no? Would you really want this in the old init.d script that you use on non-systemd machines?

Reply:
@dragon512 This is alreadyin trafficserver.in (the template for the sysvinit script) on lines 180-184

Reply:
The issue is that this is already broken. I am just trying to not have it get more broken. The values here need to be set at runtime. ideally from a value that is defined by traffic server. The group and user that is used can be defined in a config file and reset via traffic_layout when using the runroot feature. The minium Tweak here is to add the TS_USER and TS_GROUP variable in the ".service" file and some basic default in the trafficserver.in file. the Environment file is often tweaked and generated by chief for many people. Having a working service file that can be configured by default I think is the issue I am concerned about. Could you please add these variable to the setup so our users can have a better more configurable experience by default.

Reply:
The issue is that this is already broken. I am just trying to not have it get more broken. The values here need to be set at runtime. ideally from a value that is defined by traffic server. The group and user that is used can be defined in a config file and reset via traffic_layout when using the runroot feature. The minium Tweak here is to add the TS_USER and TS_GROUP variable in the ".service" file and some basic default in the trafficserver.in file. the Environment file is often tweaked and generated by chief for many people. Having a working service file that can be configured by default I think is the issue I am concerned about. Could you please add these variable to the setup so our users can have a better more configurable experience by default.